src/app/my/reviews/page.tsx

```diff
@@ -1,3 +1,77 @@
-export default function Page() {
-  return <div>나의 리뷰</div>;
+'use client';
+
+import { useInfiniteQuery } from '@tanstack/react-query';
+import { useEffect, useRef } from 'react';
+import { userQueries } from '@/api/queries/userQueries';
+import ReviewCard from '@/components/crew/ReviewCard';
+import Spinner from '@/components/ui/Spinner';
+
+export default function MyReviewsPage() {
+  const bottomRef = useRef<HTMLDivElement | null>(null);
```

Comment:
```suggestion
  const bottomRef = useInfiniteScroll(() => fetchNextPage(), hasNextPage);
```
useInfiniteScroll 훅을 사용하면 ref를 생성하지 않아도 됩니다!

순서는 useInfiniteQuery 호출 아래에 있어야 합니다.